WebContent/WEB-INF/jsp/controlPanel.jsp

```diff
@@ -143,7 +143,7 @@
 									   <span class="submenuText"><input type="radio" name="size" value="2" > Service (768px)</span><br />
 									   <span class="submenuText"><input type="radio" name="size" value="6" > Large (1200px)</span><br />
 									   <span class="submenuText"><input type="radio" name="size" value="7" > Huge (1600px)</span><br />
-									   <span class="submenuText"><input type="radio" name="size" value="v" > Videos mp4 (720p) </span><br />
+									   <span class="submenuText"><input type="radio" name="size" value="v" > MP3/MP4 </span><br />
```

Comment:
That would be great if you don't mind. Given the significance of this PR, we should keep it clean without other changes included. Thanks Longshou.